smart-contracts/deployments/ElliotsDeployment/BridgeRegistry.json

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
Did you mean to commit files in smart-contracts/deployments/ElliotsDeployment?

Reply:
Just deleted them